docs/BreakingChanges/107 Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException.md

```diff
@@ -24,7 +24,6 @@ This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgr
 ### Affected APIs
 * `E:System.Windows.Controls.DataGrid.UnloadingRow`
 * `E:System.Windows.Controls.DataGrid.UnloadingRowDetails`
-* `E:System.Windows.Controls.DataGrid.UnloadingRowGroup`
```

Comment:
Some of these APIs we listed are actually only in Silverlight or .NET Core, so are not ones we want to flag for breaking changes